lua/Application.lua

```diff
@@ -472,6 +472,8 @@ metaTableApplication_ = {
 -- "box", "default", "ellipsis", "hourglass", "poi", "reload", "ring", "ringAlt", "ripple", "rolling", "spin",
 -- "squares", "triangle", "wheel" (see http://loading.io/). The default value is 'default'.
 -- @arg data.key An optional string with 39 characters describing the Google Maps key (see https://developers.google.com/maps/documentation/javascript/get-api-key).
+-- The Google Maps API key allow monitors your Application usage in the Google API Console.
+-- This parameter is compulsory when the Application has 25,000 map loads per day, or when the Application will be installed on a server.
```

Comment:
The Google Maps API key monitors your Application's usage in the Google API Console.
 +-- This parameter is compulsory when the Application has at least 25,000 map loads per day,